spec/factories/books.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :book do
+    genre 'genre'
```

Comment:
Use faker to create the data, if you set it as strings it will always have the same. Take a look at the docs https://github.com/stympy/faker